src/module_4/inference.py

```diff
@@ -0,0 +1,26 @@
+import logging
+import os
+from joblib import load
+from train import *
+from utils import *
+
+logger = logging.getLogger(__name__)
+logger.level = logging.INFO
+
+consoleHandler = logging.StreamHandler()
+logger.addHandler(consoleHandler)
+
+def main():
+    model_name = "20240217-201103_XGBoost_0.25.pkl"
```

Comment:
This should be a config. You can set it up as a global variable but ideally this script should be called passing which model to load, right?